README.md

```diff
@@ -12,7 +12,7 @@
         <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
 </p>
 
-# Amazon_ads Source dbt Package ([Docs](https://fivetran.github.io/dbt_amazon_ads/))
```

Comment:
Let's change this to `Amazon Ads` to read a bit better. Also let's update the source package README to reflect this as well.

Reply:
done

Reply:
@fivetran-catfritz actually now that I look at this again, we say `Source`. We should remove that from the transformation package.

Reply:
Haha yeah I saw that too. Done!